comps/retrievers/langchain/pinecone/ingest.py

```diff
@@ -63,7 +63,7 @@ def ingest_documents():
     contains Edgar 10k filings data for Nike."""
     # Load list of pdfs
     company_name = "Nike"
-    data_path = "../data/"
+    data_path = "data/"
```

Comment:
This script is unused in anywhere, would you mind delete this one and also the `nke-10k-2023.pdf`.